Part_1/Pointer/777.cpp

```diff
@@ -1,28 +1,27 @@
 #include <iostream>
 
-using namespace std;
-
-void swap(int* p1, int* p2) {
+void swap(int *p1, int *p2) 
+{
     int tmp;
```

Comment:
This implementation still swaps the value of `int` variables. Can we swap the value of `int *`?